source/adapters/level_zero/v2/api.cpp

```diff
@@ -0,0 +1,1105 @@
+/*
+ *
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM
+ * Exceptions. See LICENSE.TXT SPDX-License-Identifier: Apache-2.0 WITH
+ * LLVM-exception
+ *
+ * @file ur_api.cpp
+ * @version v0.10-r0
+ *
+ */
+#include <mutex>
+#include <ur_api.h>
+#include <ze_api.h>
+
+#include "../common.hpp"
+#include "logger/ur_logger.hpp"
+
+std::mutex ZeCall::GlobalLock;
+
+ur_result_t UR_APICALL
+urContextCreate(uint32_t DeviceCount, const ur_device_handle_t *phDevices,
+                const ur_context_properties_t *pProperties,
+                ur_context_handle_t *phContext) {
+  logger::error(logger::LegacyMessage("[UR][L0] {} function not implemented!"),
```

Comment:
please remove the `logger::LegacyMessage`. This type only exists for compatibility with old PI-style logs.

Reply:
Done.